cohort-engine-api/src/main/java/com/ibm/cohort/engine/api/service/FHIRRestUtils.java

```diff
@@ -186,12 +198,45 @@ private static MeasureParameterInfo toMeasureParameterInfo(ParameterDefinition p
 		parameterDefinition.getExtension().stream()
 				.filter(MeasureEvaluationSeeder::isDefaultValue).findFirst() // only zero or one per IG
 				.map(Extension::getValue)
-				.map(Object::toString)
+				.map(x -> {if(defaultValue != null){return defaultValue;} else return x.toString();})
 				.ifPresent(retVal::setDefaultValue);
 
 		return retVal;
 	}
-	
+
+	static String complicatedTypeValueConstructor(ParameterDefinition parameterDefinition){
+		FhirContext context = FhirContext.forR4();
+		IParser parser = context.newJsonParser();
+		String retVal;
+		String intermediateValue;
+		Patient patient = new Patient();
```

Comment:
Is using `Patient` to store an extension so that you have an object compatible with `parser.encodeResourceToString(patient)` below?

Reply:
Yeah, I can add a comment making that clearer.